improver/wxcode/wxcode_decision_tree.py

```diff
@@ -126,44 +129,47 @@ def wxcode_decision_tree():
             'diagnostic_thresholds': [(0.8125, 1)],
             'diagnostic_conditions': ['above']},
 
+        # A.1.1
         'heavy_sleet_continuous': {
             'succeed': 18,
             'fail': 'heavy_rain_or_snow_continuous',
             'probability_thresholds': [0., 0.],
-            'threshold_condition': '>=',
+            'threshold_condition': '>',
             'condition_combination': 'AND',
             'diagnostic_fields':
-                [['probability_of_lwe_snowfall_rate_above_threshold',
-                  'probability_of_rainfall_rate_above_threshold'],
-                 ['probability_of_rainfall_rate_above_threshold',
-                  'probability_of_lwe_snowfall_rate_above_threshold']],
-            'diagnostic_gamma': [0.7, 1.0],
+                [['probability_of_rainfall_rate_above_threshold',
+                  'probability_of_lwe_sleetfall_rate_above_threshold'],
+                 ['probability_of_lwe_snowfall_rate_above_threshold',
+                  'probability_of_lwe_sleetfall_rate_above_threshold']],
+            'diagnostic_gamma': [1.0, 1.0],
```

Comment:
Is `diagnostic_gamma` still required here?  What's being compared exactly?

Reply:
Yes. It is reaching True if P(Sleet-rate) > P(Snow-rate) AND P(Sleet-rate) > P(rain-rate). It is doing this by taking a difference between each pair (after multiplying by the gamma) and comparing with zero. A bit convoluted, but there is (or was) another example of using it for this purpose.

Reply:
OK, that is indeed convoluted but makes sense!